ethsigner/signer/multifile-based/src/main/java/tech/pegasys/ethsigner/signer/multifilebased/KeyPasswordFile.java

```diff
@@ -22,14 +22,14 @@
   private final Path key;
   private final Path password;
 
-  KeyPasswordFile(final Path key, final Path password) {
-    this.key = key;
-    this.password = password;
+  KeyPasswordFile(final Path keyPath, final Path passwordPath) {
+    this.key = keyPath;
```

Comment:
nit: update the field names to be consistent with the constructor arg names

Reply:
done

Reply:
done